src/main/javascript/src/resources/components/footer/footer.html

```diff
@@ -0,0 +1,14 @@
+<template>
+	<div class="footer">
+		<div>
+			<a href="http://carleton.ca/privacy/policies/" target="_blank">Privacy Policies</a>
+
+			<a href="http://carleton.ca/accessibility/" target="_blank">Accessibility</a>
+
+			<a href="mailto:crmsupport@carleton.ca?Subject=Support%20Request%20or%20Feedback">Support Request/Feedback</a>
+
+		<div class="copyright">
+			Copyright - Carleton
+		</div>
+	</div>
+</template>
```

Comment:
Doesn't look like any of the requested changes were committed. You've also got an open div on line 3 with no closing div

Reply:
The footer is not really a footer at the moment.
![image](https://user-images.githubusercontent.com/32455969/78315787-06b8b300-752c-11ea-8d1d-cd38ce7afb54.png)

Reply:
My mistake, changes should be pushed now. I added some lines to the footer class that should put it at the bottom, but it doesn't appear to change the position of the footer.